tasks/tasks_softether.yml

```diff
@@ -40,7 +38,6 @@
   package: pkg="{{item}}"
   with_items: "{{packages}}"
   become: yes
-  when: vpnserver_present | failed
```

Comment:
The same, removing when does not cause packages to update, but save time on replay.